lib/streamlit/elements/image.py

```diff
@@ -386,6 +387,55 @@ def image_to_url(
         return ""
 
 
+def _check_if_svg_can_be_rendered_as_img(image: str) -> bool:
+    if image.find("xlink") != -1:
```

Comment:
nit: I think it would be slightly better to use `in` here:

```
if "xlink" not in image:
   ...
```

Some more context [here](https://stackoverflow.com/questions/18437798/find-vs-in-operation-in-string-python).

Reply:
I've added some docstring and put:
```
    if "xlink" in image:
```
there 👍

Reply:
Also I've removed unreferenced snapshot 👍